Update UG

0xteet0 opened the pull request:
Address Issue #35

Changes
- Add User Guide for marking of student attendance

0xteet0 committed:
Update UG

Add User Guide for marking of student attendance

ElginTZM commented on docs/UserGuide.md:
```diff
@@ -86,6 +89,20 @@ Examples:
 
+### Marking attendance of student: `mark`
+
+Format: `mark /name STUDENTNAME | /id STUDENTID /attendance ATTENDANCE`
```
Can mark it as [Coming Soon] as it hasn't been implemented in the current version yet.

0xteet0 replied:
Good catch! Will update it now!

ElginTZM reviewed (changes requested):
Everything looks good! 👍 Just a small issue with the feature not marked as [Coming Soon].

Edit: Resolved

0xteet0 committed:
Update UG

Update user guide for marking attendance of student to be coming soon as it has yet to be implemented in this version.

ElginTZM reviewed (approved):
LGTM! 👍
